dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -147,8 +147,19 @@ func buildModules(srcPath, name, ufsType, moduleFlag, version string, modules ma
 		for _, arg := range strings.Split(moduleEntry.mavenArgs, " ") {
 			moduleMvnArgs = append(moduleMvnArgs, arg)
 		}
+		var versionMvnArg string
```

Comment:
should this be set to the default value of `ufs.hadoop.version` within maven? What will happen if `ufs.hadoop.version` isn't overridden?

Reply:
Defaulted to 2.2.0 like it is in UFS Constants template